pkg/models/user_auth.go

```diff
@@ -100,6 +100,17 @@ type GetAuthInfoQuery struct {
 	Result *UserAuth
 }
 
+type TeamOrgGroupDTO struct {
```

Comment:
I'm not 100% sure this is the _best_ place to have these two but is _a_ place. Everything here is 

> technically

 related to LDAP authentication.

Reply:
Yea, tricky but here or in api module is fine